owiddata/generate-owiddata-stats.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import datetime
+import json
+from datetime import date
+import matplotlib.pyplot as plt
+import os
+import pandas as pd
+#import geopandas
```

Comment:
super nitpicky -- do you want to delete these if they're not being used?

Reply:
Good point! I'll take them out in case we don't end up adding figures.